examples/FirebasePush_ESP8266/FirebasePush_ESP8266.ino

```diff
@@ -37,19 +37,23 @@ void setup() {
   Serial.print("connected: ");
   Serial.println(WiFi.localIP());
 
-  // add a new entry.
-  String l = fbase.push("/logs", "{\".sv\": \"timestamp\"}");
-  // handle error.
-  if (fbase.error()) {
+   // add a new entry.
+  FirebaseResultWithMessage push = fbase.push("/logs", "{\".sv\": \"timestamp\"}");
```

Comment:
Yup, I had it then decided it wasn't worth the complexity to maintain two types. I liked that it was explicit in what returned a body and what didn't.
I will fix it.